lib/octokit/client/actions_workflow_runs.rb

```diff
@@ -0,0 +1,98 @@
+      # @option options [String] :branch Optional filtering by a branch
+      # @option options [String] :event Optional filtering by the event type
+      # @option options [String] :status Optional filtering by a status or conclusion
+      #
+      # @return [Sawyer::Resource] the total count and an array of workflows
+      # @see https://developer.github.com/v3/actions/workflow-runs/#list-workflow-runs
+      def workflow_runs(repo, workflow, options = {})
+        paginate "#{Repository.path repo}/actions/workflows/#{workflow}/runs", options
+      end
+      alias list_workflow_runs workflow_runs
+
+      # List all workflow runs for a repository
+      #
+      # @param repo [Integer, String, Repository, Hash] A GitHub repository
+      # @option options [String] :actor Optional filtering by the login of a user
+      # @option options [String] :branch Optional filtering by a branch
+      # @option options [String] :event Optional filtering by the event type (e.g. push, pull_request, issue)
+      # @option options [String] :status Optional filtering by a status or conclusion (e.g. success, completed...)
+      #
+      # @return [Sawyer::Resource] the total count and an array of workflows
+      # @see https://developer.github.com/v3/actions/workflow-runs/#list-repository-workflow-runs
+      def repository_workflow_runs(repo, options = {})
+        paginate "#{Repository.path repo}/actions/runs", options
+      end
+      alias list_repository_workflow_runs repository_workflow_runs
+
+      # Get a workflow run
+      #
+      # @param repo [Integer, String, Repository, Hash] A GitHub repository
+      # @param id [Integer] Id of a workflow run
+      #
+      # @return [Sawyer::Resource] Run information
+      # @see https://developer.github.com/v3/actions/workflow-runs/#get-a-workflow-run
+      def workflow_run(repo, id, options = {})
+        get "#{Repository.path repo}/actions/runs/#{id}", options
+      end
+
+      # Re-runs a workflow run
+      #
+      # @param repo [Integer, String, Repository, Hash] A GitHub repository
+      # @param id [Integer] Id of a workflow run
+      #
+      # @return [Boolean] Returns true if the re-run request was accepted
+      # @see https://developer.github.com/v3/actions/workflow-runs/#re-run-a-workflow
+      def rerun_workflow_run(repo, id, options = {})
+        post "#{Repository.path repo}/actions/runs/#{id}/rerun", options
+
+        last_response.status == 201
+      end
+
+      # Cancels a workflow run
+      #
+      # @param repo [Integer, String, Repository, Hash] A GitHub repository
+      # @param id [Integer] Id of a workflow run
+      #
+      # @return [Boolean] Returns true if the cancellation was accepted
+      # @see https://developer.github.com/v3/actions/workflow-runs/#cancel-a-workflow-run
+      def cancel_workflow_run(repo, id, options = {})
+        post "#{Repository.path repo}/actions/runs/#{id}/cancel", options
+
+        last_response.status == 202
```

Comment:
Do we want to add the `202` status as an option for `boolean_from_response` so that we could condense this?

```suggestion
        boolean_from_response :post, "#{Repository.path repo}/actions/runs/#{id}/cancel", options
```

Reply:
Hey @tarebyte, thanks for the review :tada:!

I was thinking about making them using `boolean_from_response`. My reasoning to change it this way was because I've seen that boolean_from_response handles `Octokit::NotFound` as false. 

`cancel_workflow_run` raises `Octokit::Conflict` if the run is already completed. `rerun_workflow_run` raises `Octokit::Forbidden` if the run cannot be rerun. Not sure if I have to handle those exception in `boolean_from_response` in like it handles `Octokit::NotFound` and return false in those events.

I think it is going to be useful for the developers to leave those exceptions unhandled because they carry useful information about why the request was not executed correctly.

What do you think, should I handle those exceptions as falsey response or leave them as-is?

Reply:
> What do you think, should I handle those exceptions as falsey response or leave them unhandled?

I agree the errors here are really useful, I don't think we should rescue them like we do `Octokit::NotFound`.

Reply:
Thanks for the fast response! I've added a commit that addresses the discussion above. :)